Oshan Medbay Remodel

TheGeneralJay opened the pull request:
<!-- The text between the arrows are comments - they won't be visible on your PR. -->
<!-- To label this PR, add the label(s) without the prefixes surrounded by brackets anywhere, for example: [LABEL] -->
<!-- PRs should at least have one area (A-) label and at least one category (C-) label -->
[Mapping] [QoL]

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Attempts to address common problems that players have with Oshan medbay. Provides genetics with hall-adjacent windows and shifts the therapy office into its own room with a door. Generally seeks to provide room for crime via maintenance doors at the end of the hall now instead of an open-concept therapy office, and places genetics closer to the booth in general. The morgue has also been made larger to account for the larger population requirement when rolling Oshan and the inevitable trench deaths that follow, giving a bit more room for doctors to work.

<img width="1187" height="860" alt="{78665A0D-AE7C-4FFE-88BB-2353111027CE}" src="https://github.com/user-attachments/assets/acefadd6-725b-4624-bb4b-932e35914901" />

## Why's this needed? <!-- Describe why you think this should be added to the game. -->
I've heard many complaints that Oshan genetics can be difficult to get volunteers for, sometimes merely because it just isn't near the main hall. This places it where the pharmacy used to be, allowing for people to ask the geneticists for genes more directly without entering medbay as a whole. This also moves the monkey dome, which was always used as a path for Beepsky and ended up loosing the monkeys upon unsuspecting doctors and their clean tables.

Additionally, this does away with the open concept therapy office in exchange for one closed off with blinds and a maintenance door. This follows the formula for how therapy offices are set up on other maps, providing ample space for therapy RP, scheming, or crime.

In general, Oshan medbay is good, but I think it benefits from moving some rooms around.

## Testing <!-- Please provide at least one screenshot of your changes working if appropriate, or a description of how you've tested them if not. -->
Ran the changes and ensured all APCs are hooked to the main grid, disposal chutes lead to disposals, and mail chutes are functional. Gave a few run-throughs to ensure lighting is operational and camera coverage should be suitable. If more (or less) camera coverage is required just lmk!

Screenshots of the working map:
<img width="825" height="793" alt="{C6D703F4-1EA5-404E-B3C1-410C47EC3A17}" src="https://github.com/user-attachments/assets/ba92a0fb-de06-46ba-bfc2-6ab5a8568e52" />

<img width="615" height="554" alt="{12BD88C9-8A80-4D4C-AF4F-7D729D6D0208}" src="https://github.com/user-attachments/assets/4a8086c6-09a9-445e-b705-0bdf36cbac87" />

<img width="862" height="542" alt="{80C3EDD5-3028-4FB1-8BB1-834D9B710DBB}" src="https://github.com/user-attachments/assets/ba98e1b2-8e0f-48a1-9981-a6fde4d13316" />

<img width="544" height="463" alt="{9D05D074-DDEC-4988-99CB-D4E9C9B898A4}" src="https://github.com/user-attachments/assets/4e72b408-b535-421b-a76f-8dc63869f2ec" />

<img width="513" height="550" alt="{1B61EBC0-75EF-472B-A5C5-4261146549AA}" src="https://github.com/user-attachments/assets/c2493a2a-2340-45e1-9e4a-6b97f0a8a0b8" />

Sorry about the double post, I really shouldn't be working with PRs before I get my coffee into me.

## Changelog <!-- If necessary, put your changelog entry below. Otherwise, /please/ delete this entire section. -->

<!-- Put how you want to be credited in the changelog in place of CodeDude. -->
<!-- Use (*) for major changes and (+) for minor changes. See the contributor guide for details. For example: -->

```changelog
(u)TheGeneralJay
(*)Oshan medbay has had a remodel! Details to follow in minor changes.
(+)Oshan genetics is now visible from the main hall. The pharmacy, treatment rooms, and therapy office has been moved to accommodate this.
```

JORJ949 commented:
<img width="373" height="243" alt="image" src="https://github.com/user-attachments/assets/99813489-1c6a-457a-b250-69fa46ca215e" />

IMO these should be normal walls, psych office isn't a particularly secure area and normal walls could be made fake walls easier as a path into the morgue to steal bodies

TheGeneralJay commented:
> <img alt="image" width="373" height="243" src="https://private-user-images.githubusercontent.com/159719201/489633016-99813489-1c6a-457a-b250-69fa46ca215e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTc5NjUwMzEsIm5iZiI6MTc1Nzk2NDczMSwicGF0aCI6Ii8xNTk3MTkyMDEvNDg5NjMzMDE2LTk5ODEzNDg5LTFjNmEtNDU3YS1iMjUwLTY5ZmE0NmNhMjE1ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxNVQxOTMyMTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MTYzZWI5ZjFmODIxMGQ0OGUwNTg3OTVjOTE4OTEwZmUzNWIxZDNkNWJjMGY0MDZkODU4MGE1NzU5NjMwOTRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gDDE12MGTbA5SVAwxaGDHJPVljREkIBV_4MpXqUplVY">
> IMO these should be normal walls, psych office isn't a particularly secure area and normal walls could be made fake walls easier as a path into the morgue to steal bodies

Totally fair! I'll pop those in, I agree.

aaaaaaaaaaaaaaaaaah commented:
I already prefer this to the current layout, one thing I am missing though is to finally add a second surgery table in the surgery room since there is definetly room in there and you already have two tables. 
Might also be a good idea to post this on the forums to get more player feedback.

TheGeneralJay commented:
> I already prefer this to the current layout, one thing I am missing though is to finally add a second surgery table in the surgery room since there is definetly room in there and you already have two tables. Might also be a good idea to post this on the forums to get more player feedback.

Definitely agree on the operating tables, I'm surprised I forgot to tweak that myself! I've added that and moved some stuff around, as well as adding tables for the mounted defibs as... it seemed a little odd to just have them connected to the operating table itself. With the movement, I also adjusted a portion of robotics (just the positioning of chargers, fabricators, and cargo pad) to allow for a minor expansion to keep things feeling roomy and less thrown together in medbay as a whole. Screenshots of the working changes to follow:

<img width="634" height="514" alt="{95AFA68F-A16E-419B-ADA8-98D5C36AB09F}" src="https://github.com/user-attachments/assets/c64cb19f-8534-48f4-8113-b59670de0070" />

<img width="673" height="575" alt="{ABD2A78A-6D70-4064-9FD0-624D049A21D3}" src="https://github.com/user-attachments/assets/90f2a212-8cb5-42ba-9abc-f588055dd36d" />

Egregorious commented:
This looks great. 

Maybe the little western corridor could be open, or not locked behind med access. There's no immediate access to any other medical stuff there, and I feel like it would make sense that people could visit the therapist's office without being blocked by the entrance to a lobby area.

TheGeneralJay commented:
> This looks great.
> 
> Maybe the little western corridor could be open, or not locked behind med access. There's no immediate access to any other medical stuff there, and I feel like it would make sense that people could visit the therapist's office without being blocked by the entrance to a lobby area.

Makes sense to me! I've left the doors in but removed the access, so it's at least a little bit of flooding protection for the inevitable breaches to come during rounds. I've also swapped the maintenance doors to just be regular maintenance access there, so it can also be used as an alternate path straight up to north maintenance for crime or bored exploration.

xenomni-fen commented:
IDK if its too late to suggest anything but the window to the hall at genetics seems a bit bare- maybe could do one or two roll down windows with the credit device like nadir has?

TheGeneralJay commented:
> IDK if its too late to suggest anything but the window to the hall at genetics seems a bit bare- maybe could do one or two roll down windows with the credit device like nadir has?

Not too late! I like this suggestion. I've provided geneticists with a fancy new table window to provide some variation against that stretch (and to give their happy monkey customers a fun vault area to leave).

<img width="513" height="494" alt="{92CF51A1-AF4F-49D3-BCD0-01B958412BE1}" src="https://github.com/user-attachments/assets/65d2c85f-4bba-4e97-ab82-fe721c7cfa29" />

Annnd to address Sord:

> geneticists dont deserve blinds

As for this one, I'm actually not sure if you actually don't want blinds in genetics. They have it anyways on Oshan so I carried it over to this new location. Can be removed if this is actual criticism! Lmk

Nexusuxen commented:
Big fan of moving Genetics to the front. I would like to make sure that you can see and hear gene booth customers from one or both seats, but that isn't a huge deal honestly (I get immense validation from interacting with booth customers)

Also you could always pull a Neon and give them a broken blinds button

TobleroneSwordfish reviewed (approved):
I like this, genetics makes much more sense being by a main hallway and the psych office is a good private area for crime or RP.

JORJ949 reviewed (approved):
I like this a lot, pharmacy being further back and having blinds makes it way more crimeable, as is stealing corpses thanks to the normal walls to the morgue. Also big fan of the public front door to the genelab, will make genetecists interact with crew more.

TheGeneralJay commented:
All of these suggestions have been worked into the map now. In order to make the northern maintenance area a little less barren, I have provided some wider portions where things got tight (and adjusted as necessary elsewhere), added random junk spawns and other misc. maintenance items, and I've added a hangar for medical. This makes sense to me considering every department on this map EXCEPT medical seems to have a hangar, and it provides a little extra utility to the upper portion. It mimics security's hangar on the south.

<img width="486" height="362" alt="{0E8EE939-AE57-464B-8491-2106FB604BF9}" src="https://github.com/user-attachments/assets/77f560ec-0514-4b8b-a9cf-64e3c2b61c72" />

<img width="1230" height="681" alt="{C9E6AF18-FEC4-4D63-BFBA-D94F489A4344}" src="https://github.com/user-attachments/assets/b07020d0-c17d-429a-a59a-f027cb233231" />

<img width="973" height="501" alt="{89B61C9C-0CD6-44E9-9676-6F1B35C3F8CC}" src="https://github.com/user-attachments/assets/631f7ff7-6085-49f3-abd4-8a7ceb34b518" />

I have also switched up a bit in the lobby area! The gene booth has been moved closer to the genetics window and I've added a few chairs for waiting near cloning.

<img width="1044" height="724" alt="{F72BC78D-D4AC-4AC9-A1BE-73AA12BA3ACC}" src="https://github.com/user-attachments/assets/05bc9c6c-cda4-4f71-b7a6-cc5723c43b1c" />

I have NO idea why the runtime checkers are mad about Donut2 and random rooms, but I digress. Lmk if I need to do anything to fix those issues.

Sord213 commented:
Random room/donut 2 is my fault, shouuuld be fixed
